awsenergylabelercli/awsenergylabelercli.py

```diff
@@ -126,36 +133,37 @@ def get_arguments():
     account_list.add_argument('--allowed-account-ids',
                               '-a',
                               nargs='*',
-                              default=None,
+                              default=os.environ.get('ALLOWED_ACCOUNT_IDS'),
```

Comment:
This is also a list and would require casting.

Reply:
This is covered with lines 198-204, which utilizes validators from the library (which also already check mutually exclusivity)